vendors/pyproject.toml

```diff
@@ -23,5 +23,6 @@ python = "^3.6"
 
 jsonschema = "^3.2.0"
 lark-parser = "^0.9.0"
-packaging = "^20.1"
+packaging = "^20.8"
+pyrsistent = "^0.16.0"
```

Comment:
Why is `pyrsistent` added explicitly here?

Reply:
@sdispater 'cause in ^0.17 they dropped Python 2.x support (ref: https://github.com/tobgu/pyrsistent/blob/master/CHANGES.txt#L17#L19)

Reply:
@gi0baro Thanks for the information 👍